apps/api/src/app/notification-template/notification-template.controller.ts

```diff
@@ -56,12 +59,17 @@ export class NotificationTemplateController {
     summary: 'Get notification templates',
   })
   @ExternalApiAccessible()
-  getNotificationTemplates(@UserSession() user: IJwtPayload): Promise<NotificationTemplateResponse[]> {
+  getNotificationTemplates(
+    @UserSession() user: IJwtPayload,
+    @Query() query: NotificationTemplatesRequestDto
+  ): Promise<NotificationTemplatesResponseDto> {
```

Comment:
This is amazing, could we also add a e2e API tests for the behaviour of pagination ?